ci-operator/jobs/openshift/origin/openshift-origin-release-3.11-presubmits.yaml

```diff
@@ -378,11 +379,16 @@ presubmits:
   - agent: jenkins
     always_run: true
     branches:
-    - release-3.11
-    - release-3.10
-    - release-3.9
-    - release-3.8
-    - release-3.7
+    - ^release-3\.10$
```

Comment:
I think the order matters.
https://github.com/openshift/release/blob/ee08fb89d22ca1c36b9ac360232123eb24804e1c/hack/validate-prow-job-semantics.py#L219

Reply:
yeah

I've just pwned that job, we should not have jobs that trigger on multiple different branches and 3.10 is outdated anyway.